pkg/provider/github/parse_payload.go

```diff
@@ -14,15 +14,14 @@ import (
 	ghinstallation "github.com/bradleyfalzon/ghinstallation/v2"
 	oGitHub "github.com/google/go-github/v60/github"
 	"github.com/google/go-github/v61/github"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
-
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/apis/pipelinesascode/keys"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/opscomments"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/params"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/params/info"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/params/triggertype"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/provider"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
metav1 is everywhere in our code so let's keep this (and won't conflict with auto import fo rother v1 stuff liek core or tekton etc)

Reply:
it's a refactoring! 

v1 usually clash with other namespaces and we usually use metav1 corev1 or tektonv1 across the code

Reply:
:+1: 
Updated